pkg/aiven/adapter.go

```diff
@@ -447,6 +468,14 @@ func AivenCollectors(adapter *AivenPostgreSQLAdapter) []agent.MetricCollector {
 			),
 		},
 	}
+	if adapter.PGVersion >= "17" {
```

Comment:
We can't do naive string comparisons for versions as `"17.11" < "17.2"` by string comparison rules.

Also, `"9" > "17"`